### PR TITLE
Fixes to PR655 use of bytes under Python3 and install dir for Win64

### PR DESCRIPTION
**Pull request by peircej**

### CODE REVIEW

**Comment by peircej** [comment]
This pull request **fixes 2 alerts** when merging 8cd75b8072f9b48c3d9ea3965217c5df892a2c31 into 14088062c579e4a15b76198ec83c22849e8188b2 - [view on LGTM.com](https://lgtm.com/projects/g/psychopy/psychopy/rev/pr-5d8a1112bf71a6a49a8adbe135e3a2243acd71e0)

**fixed alerts:**

* 2 for \_\_init\_\_ method calls overridden method

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: psychopy/hardware/pr.py [hunk 1]
```diff
@@ -356,55 +365,24 @@ def __del__(self):
...
-        if message[-1] != '\n':
-            message += '\n'  # append a newline if necess
-
-        # flush the read buffer first
-        # read as many chars as are in the buffer
-        self.com.read(self.com.inWaiting())
-
-        # send the message
-        for letter in message:
-            # for PR655 have to send individual chars ! :-/
```
Comment (da5nsy) [inline psychopy/hardware/pr.py:390]: Should this have been removed? It looks like it causes [this](https://discourse.psychopy.org/t/cannot-connect-photoresearch-pr655-to-pc-using-psychopy-communication-protocol-problem/25595/12) issue.

Comment (da5nsy) [reply psychopy/hardware/pr.py:390]: Tagging @peircej

Comment (da5nsy) [reply psychopy/hardware/pr.py:390]: This is reported as being fixed, here: https://github.com/psychopy/psychopy/issues/5237 / https://github.com/psychopy/psychopy-photoresearch/commit/872718f3198f317c3dc3c9dc939b8c2f6d1bec32

**Pull request merged**